src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -49,6 +49,11 @@
                 }
             ]
         },
+        {
+            "name": "vscode_autoUpdateBeta",
```

Comment:
it's not really vscode-specific, right? Could potentially be useful for e.g. VS, so `aws_` prefix might make sense?